README.adoc

```diff
@@ -78,7 +78,7 @@ The git client plugin can install link:https://git-scm.com/download/win[Git for
 * Upload that zip file to an HTTP server
 * Set the `Download URL for binary archive` as the URL of the uploaded zip file
 * Leave `Path to Git` executable as `git`
-* Specify `PortableGit-a.bb.c\bin\git.exe` for the `Subdirectory of extracted archive
+* Specify `PortableGit-a.bb.c\bin\git.exe` for the `Subdirectory of extracted archive`.  This points to the git.exe in the archive relative to the root of the archive.
```

Comment:
Thanks very much for the clarification and for your patience with my adaptations!

I think we should also include a suggestion in one of the earlier steps about the assumed contents of the zip file created by the user.
```suggestion
* Specify `PortableGit-a.bb.c\bin\git.exe` for the `Subdirectory of extracted archive`.
  This points to the git.exe in the archive relative to the root of the archive.
  This assumes the `bin` directory is stored in the archive as `PortableGit-a.bb.c\bin`
```

Reply:
The first additional line was added, the second clarifying assumptions about the 'bin' directory is likely not needed.